components/slider/__tests__/index.test.js

```diff
@@ -10,64 +10,23 @@ import { sleep } from '../../../tests/utils';
 describe('Slider', () => {
   mountTest(Slider);
   rtlTest(Slider);
-  focusTest(Slider);
+  focusTest(Slider, true, false);
```

Comment:
```suggestion
  focusTest(Slider, {
    xx: true,
    yy: false,
  });
```

要改成这样的写法了，不然多个 boolean 可读性很差。